README.md

```diff
@@ -112,6 +112,12 @@ custom:
     services:
       - kms
       - secretsmanager
+
+    # optional
+    # can pick one of subnet groups in (rds / redshift / elasticache / dax)
+    # ex) DAX is not available in ap-northeast-2 so I don't want to make DAX subnet group
+    subnetGroup:
```

Comment:
let's call this `subnetGroups` since it can represent more than one.  By default, if not specified, all of the subnet groups will be created.

Reply:
ok. I changed option name to `subnetGroups` and add some comment as you suggested